### DISCUSSION TITLE
How relevant is the boundary of ISV in the research question?

### DISCUSSION THREAD

**Discussion by JRBliekendaal** [JRBliekendaal/master-thesis#43] [Q&A] [answered]
@edzob working on the introduction. Hope to finish this (for the 10 pager) today or somewhere tomorrow. And I have a small question.

As you know the longer you work on the subject the more insights you will get. And I am doubting if the lens of an ISV is really relevant. My current research question is that of:

**_What are, __for an ISV__, the success factors of EA for antifragility in the public sector market?_**

The ISV I am talking about is part of the Public Sector Market (not the Public Sector). So why use the boundary (lens) of ISV? The outcome should be usable for all parties in the Public Sector. (no I am not loosing the public sector because then I have a complete different research.

So should it not be:

**_What are the success factors of EA for antifragility in the public sector market?_**

In this way I can also ask myself (subquestion) what is the public sector market ;) and I can explain that it is a system of systems.

The new question also fits better with my conceptual model.

![conceptualmodel](https://user-images.githubusercontent.com/86345975/134013399-063a834e-be0b-46f8-8ae8-67eaed454841.png)

I have the feeling that adding ISV as a lens it will not make a difference (unless I go for IT Systems, which I do not want to do).

**Comment by JRBliekendaal** [comment]
I also send an email to Hans on this post.

**Comment by edzob** [comment]
>> What are, for an ISV, the success factors of EA for antifragility in the public sector market?
>> What are the success factors of EA for antifragility in the public sector market?

The delta beteen the two question is the rol/actor of the ISV (think demo patterns).
When you include a role or actor then they take roles. The roles have relation to the service and the resonsbility.
When you identity 1 actor you need to at least idientify 1 other actor, otherwise there is no transaction (service/product delivered)

In both cases you need to define which EA and which stakeholders are relevant (see EA definition in my thesis).
Are you addressing the NORA? and all stakeholders? why should stakeholders apply to one EA defintion? is this a correct EA def or only in name only?

The  Public Sector Market is large.
In both cases you need to identify the sub-set (gemeente, provincie, landelijk, ZBO's etc)
in both cases you need to identify the stakeholders relevant to the EA and the E in the EA.
in both cases you need to identify "the system" that needs to be antifragile and then think about the System-of-Systems application.

When you do this then the question might become more clear to you.
Iterations around the problem domain, the research question and the body of knowledge :)

Does this help?

**Reply by JRBliekendaal** [reply]
No not really 🤣 I know when I follow demo I will get a complete other research question. Simply because public sector is already a composite. And public sector market is not even een actor but a boundary. 

If something is everything than something is nothing :)

I want to take a look not at EA but how EA should be for antifragile. Or the success factors of EA to steer towards antifragile. Not only usable for an ISV (already invited a lot of members of the market itself for validation) but generalised. Even have some interviews with 'bestuurders' to validate the issue of VUCA in the public sector. 

For me the EA 'function' is already an composite. If you take a look at the specific properties of the public sector (a lot of research on this one) you could confront this with the EA function and the success factors for Antifragile. 

I just go on writing and will think about it in the mean time. ;) I still have a couple of months on how to deal with this one. 

Btw I defined the public sector market and the ISV as boundaries. Not as concepts ;) the concepts are simple. EA and Antifragile with success factors as mediator. 🤣

**Reply by edzob** [reply]
Defining what the EA is (a capbility, a function, something else) is a complex beast.
Defining who is responsbile of the EA is also a complex beast.
See for example the role of EA in the causal-loop diagram in our DYA  whitepaper. 
https://labs.sogeti.com/wp-content/uploads/2020/09/Value-sensitive-architecture-20200902-v2.pdf

In your thesis it should be clear which lens you apply to the concept "EA" 
because then you can translate the "attributes" that need to be there.

It might be a nice excersise to write or draw for me what you see as EA in your mental model, and how you see the adaption of this towards AF and what this then implies for the varous actors in this context.

**Comment by JRBliekendaal** [comment]
@edzob Nice that you mention NORA. To be honest. I work a lot with NORA but also GEMMA, PETRA, WILMA and others. Those reference architectures are all about control. I do not believe that you have to be in control of everything. As you already told with the example of the barbbel it could work for you stable part (or your residual if you take O'Reilly (2020)) but I get the feeling more and more that emerging Enterprise Architecture is a better way of doing things instead of intentional architecture. Intentional should only be reserved for the residual parts.

So it could be that I dare to state that all the current reference architectures are not suitable for a antifragile world :D (moving onto a slippery slope).

**Comment by edzob** [comment]
>> but I get the feeling more and more that emerging Enterprise Architecture is a better way of doing things 
>> instead of intentional architecture

I agree, do know that other will play a semantics game.
When you work via principles, or when you work with "game theory" then you also apply a type of control that enables emergent behaviour.

> So it could be that I dare to state that all the current reference architectures are not suitable for a antifragile world 
well yes and no
Going back to actors and roles.
in the HORA some business functions (bedrijfsfuncties) are identified and some actors/roles/stakeholders are identified.
I think this can be very powerfull..

Basically it is the balance between insert-stress, monotinicity and the shared vision and mental model from the EAAL.

How can you learn as a group if you do not know of you have a shared model?

**Comment by JRBliekendaal** [comment, marked as answer]
@edzob btw had answers from Hans. From his point of view it comes back to the question we also talked about in our meeting in front of the AMS. Sorry for Dutch.

`Hoi René,

Leuke discussie, die wij met ons drieën ook voerde op het terras. Beide opties zijn mogelijk namelijk de impact van antifragility op EA of zoals hieronder de contributie van EA op het realiseren van een Antifragiele organisatie. 

Ik denk dat het goed is dat wij in ons volgende overleg de knoop doorhakken.

Tot vrijdag, daar zie ik naar uit,

Hans`

Ik had zelf die knoop al doorgehakt.

`Hans

Ik werk nog steeds toe naar de contributie van EA op het realiseren van antifragiliteit binnen een systeem (kan idd organisatie zijn maar ook de publieke markt met haar eigen eigenschappen (system of systems) hier is heel veel onderzoek naar gedaan) Ik haal dit in de inleiding ook diverse malen aan. 

Daarom ook mijn vraag. Hoe relevant is dan de scoping op een ISV. Scoping op de public sector markt ja. Is een specifiek systeem. Dat heb je vorige week weer meegemaakt. Plannen maken zonder de impact te bepalen. Heerlijk droogzwemmen zonder toetsing. 

Volgens mij is ISV niet relevant voor de onderzoeksvraag en de scoping. Is alleen contextueel omdat 1 ik er bij werk en 2 het een onderdeel kan zijn van de publieke sector markt. Zoals het er nu staat lees ik dat het om EA van een ISV gaat om antifragile te worden in de publieke sector.

Dus niet

What are, for an ISV, the success factors of EA for antifragility in the public sector market?

Maar

What are the success factors of EA for antifragility in the public sector market?


Ik heb voor mijzelf de knoop al doorgehakt. Maar wil het nog even verifiëren bij jou en Edzo. Edzo ziet het op dit moment als een performa transactie. Twee actoren nodig 🤣 dus ISV en publieke sector als de twee CA’s. Maar ik wil niet vanuit een ISV gaan beredeneren en zeker niet in een transactie. Dan is de uitkomst niet heel bruikbaar in de publieke sector. Het is naar mijn mening dan te eenzijdig. 

Ik verwacht dan ook veel minder deelname in de Delphi groep. Een gemeente gaat niet zomaar meewerken om een ISV beter te maken. En valideren met alleen concurrenten is ook niet zaligmakend als ze al mee willen doen. Scope het op de publieke markt dan begint iedereen te rennen 😬 en dan is de impact ook het grootst.

Mvgrt

René `


En later nog:

`Hoi René,

Je was nog laat bezig!

ISV kan ook in de scoping van de onderzoeksaanpak terugkomen, bijvoorbeeld in de keuzes van de gevalsstudies en/of populatie van de respondenten/participanten.

Dus hak de knoop maar door :-)

Tot vrijdag,

Hans`

Dus voor nu ga ik even uit van de onderzoeksvraag zonder ISV. Ik laat hem wel terug komen in de delphi groep. Zal hem hier ook nog iets verder scopen op de lokale overheden (andere dynamiek).

Even uitleg over vorige week. Ik was physiek op het iBestuur congres rondom de i-Strategie van de overheid. Hans was daar ook (op zijn fiets).

We bespreken het gezamelijk op 4 oktober (status meeting).

**Reply by edzob** [reply]
>> Dus voor nu ga ik even uit van de onderzoeksvraag zonder ISV. Ik laat hem wel terug komen in de delphi groep.
Check. 
Ben benieuwd hoe de uiteindelijke vraag wordt geframed zodat er door de deelnemers vanuit een shared-mental-model geredeneerd kan worden.

**Reply by JRBliekendaal** [reply]
Daar gaan we aan werken. Doel voormijzelf is om eerst even een concept van het geheel op papier te hebben. Ben nu te circulair bezig. Dan kunnen we op basis van het concept het gaan fijnslijpen in iteraties ;)

Heb het idem gedaan met mijn Bachelor... achter aan een aanpassing en dan naar voren werken om het weer goed te krijgen. Werkt ideaal. Het wordt dan steeds scherper.

**Reply by JRBliekendaal** [reply]
The real framing comes out of chapter 2. There I will research the concept and the choices I have to make. The real mental model comes out of the analysis of theory
